client/dist/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>CAPBNB photos service</title>
+    <link rel="stylesheet" href="./styles.css" type="text/css">
+    <script src="https://code.jquery.com/jquery-3.4.1.js"
```

Comment:
In `package.json`:

`npm install --save jquery`

In files making AJAX requests, at the top:

`import $ from 'jquery'`